runtime/common/src/fee.rs

```diff
@@ -70,14 +70,16 @@ mod test {
 		assert!(x.max(y) - x.min(y) < MILLICENTS);
 	}
 
-	#[test]
-	fn check_weight() {
-		let p = CENTS / 10;
-		let q = Balance::from(ExtrinsicBaseWeight::get().ref_time());
+	// #[test]
```

Comment:
Why is this test commented out? Should it be removed?

Reply:
Good question. I am working on getting the new values. The improvements in weights are causing this test to fail.

Reply:
I wanted to follow up regarding the test we implemented to monitor improvements or regressions in the extrinsic weights. After reviewing it, I have updated the test to reflect the new range of weights based on the improvements observed.

We are not planning to make any changes to our fees at this time. However, please feel free to share your opinion.